<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservaWidget</class>
 <widget class="QWidget" name="ReservaWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="formGroupBox">
     <property name="title">
      <string>Nueva Reserva</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="vehicleLabel">
        <property name="text">
         <string>Veh√≠culo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="vehicleComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="startDateLabel">
        <property name="text">
         <string>Fecha inicio</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="startDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="endDateLabel">
        <property name="text">
         <string>Fecha fin</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="endDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="insuranceCheckBox">
        <property name="text">
         <string>Seguro</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="nuevaReservaButton">
          <property name="text">
           <string>Nueva reserva</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelReservaButton">
          <property name="text">
           <string>Cancelar reserva</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="reservationsTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
